x-pack/plugins/security_solution/scripts/extract_tactics_techniques_mitre.js

```diff
@@ -9,6 +9,7 @@ require('../../../../src/setup_node_env');
 const fs = require('fs');
 // eslint-disable-next-line import/no-extraneous-dependencies
 const fetch = require('node-fetch');
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I wouldn't think this is necessary. It should correctly get this from the root package.json file right?

Reply:
@watson without that eslint fails. I think it is the same case as `node-fetch`

Reply:
@watson we talked about this offline. It's not a problem with the eslintcache. I'm going to resolve the conversation and left it as is. I think it is the same problem we have on that file with `node-fetch`.